interfaces/builtin/unity8_contacts.go

```diff
@@ -0,0 +1,182 @@
+	interface=org.freedesktop.DBus.*,
+
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/**
+	interface=org.gnome.evolution.dataserver.AddressBook,
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/**
+	interface=org.gnome.evolution.dataserver.DirectBook,
+dbus (receive, send)
+	bus=session
+	path=/org/gnome/evolution/dataserver/Subprocess/**
+	interface=org.freedesktop.DBus.*,
+
+########################
+# Canonical - AddressBook
+########################
+dbus (receive, send)
+	bus=session
+	path=/com/canonical/pim/AddressBook,
+dbus (receive, send)
+	bus=session
+	path=/com/canonical/pim/AddressBook
+	interface=com.canonical.pim.AddressBook,
+dbus (receive, send)
+	bus=session
+    path=/com/canonical/pim/AddressBook
+	interface=org.freedesktop.DBus.*,
+
+dbus (receive, send)
+	bus=session
+	path=/com/canonical/pim/AddressBookView,
+dbus (receive, send)
+	bus=session
+	path=/com/canonical/pim/AddressBookView
+	interface=com.canonical.pim.AddressBookView,
+dbus (receive, send)
+	bus=session
+    path=/com/canonical/pim/AddressBookView
+	interface=org.freedesktop.DBus.*,
+`
+var unity8ContactsConnectedPlugAppArmor = `
+# Description: Can access contacts. This policy group is reserved for vetted
+#  applications only in this version of the policy. Once LP: #1227821 is
+#  fixed, this can be moved out of reserved status.
+# Usage: reserved
+
+dbus (receive, send)
+     bus=session
+     path=/org/gnome/evolution/dataserver/AddressBookFactory
+     peer=(label=unconfined),
+dbus (receive, send)
+     bus=session
+     path=/org/gnome/evolution/dataserver/Subprocess/**
+     peer=(label=unconfined),
+dbus (receive, send)
+     bus=session
+     path=/org/gnome/evolution/dataserver/AddressBookView/**
+     peer=(label=unconfined),
+
+dbus (receive, send)
+     bus=session
+     path=/com/canonical/pim/AddressBook
+     peer=(label=unconfined),
+dbus (receive, send)
+     bus=session
+     path=/com/canonical/pim/AddressBookView/**
+     peer=(label=unconfined),
+
+
+# LP: #1319546. Apps shouldn't talk directly to sync-monitor, but allow it for
+# now for trusted apps until buteo is integrated with push
+# notifications.
+dbus (receive, send)
+     bus=session
+     path=/synchronizer{,/**}
+     peer=(label=unconfined),
+`
+
+var unity8ContactsPermanentSlotDBus = `
+	<allow own="org.gnome.evolution.dataserver.AddressBook9"/>
+	<allow send_destination="org.gnome.evolution.dataserver.AddressBook9"/>
+	<allow send_interface="org.gnome.evolution.dataserver.AddressBook"/>
+	<allow send_interface="org.gnome.evolution.dataserver.AddressBookView"/>
+	<allow send_interface="org.gnome.evolution.dataserver.AddressBookFactory"/>
+
+	<allow own="com.canonical.pim"/>
+	<allow send_destination="com.canonical.pim"/>
+	<allow send_destination="com.canonical.pim.AddressBook"/>
+	<allow send_destination="com.canonical.pim.AddressBookView"/>
+`
```

Comment:
Everything I said about the calendar interface applies to the contacts interface (backend is not contacts-specific, use ###PLUG_SECURITY_TAGS### in ConnectedSlot, update PermanentSlot, update ConnectedPlug for classic and ###SLOT_SECURITY_TAGS###, and dbus is problematic.